test/Microsoft.Health.Fhir.Ingest.UnitTests/Microsoft.Health.Fhir.Ingest.UnitTests.csproj

```diff
@@ -39,42 +39,9 @@
     <ProjectReference Include="..\Microsoft.Health.Tests.Common\Microsoft.Health.Tests.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <None Update="TestInput\data_CodeValueFhirTemplate_Components.json">

```

Comment:
Was this change intentional? If so, should the other .json files not be copied to the output directory?

Reply:
These tests are moved along with the source code to a new assembly.